projects/helper/env.js

```diff
@@ -14,6 +14,7 @@ const DEFAULTS = {
   RENEC_RPC: "https://api-mainnet-beta.renec.foundation:8899/",
   BSQUARED_RPC: "https://rpc.bsquared.network,https://b2-mainnet.alt.technology", // add manually, short name is b2-mainnet
   BTR_RPC_MULTICALL: '0xc8818aaeaBF0dF9f3f3ffF54Ab185705177A6234',
+  BOUNCEBIT_RPC: "https://fullnode-mainnet.bouncebitapi.com"
```

Comment:
this rpc is down now?
what is backing the bbusd?
also, can you share the bitswap website and factory address?

Reply:
1. We provide a defilama specific RPC in the TG group

2. BBUSD are wrapped USDT from ETH and BNB chain.  Wrapping occurs on the respective chains and then bridged to BounceBit Chain.  Assets are store in CEFFU custody to allow CEX mirroring.  Methodology is similar to Ethena.

3. Bitswap URL: https://app.bouncebit.io/club/1
and factory address
v3: 0x30a326d09E01d7960a0A2639c8F13362e6cd304A
v2: 0x6d2Ae8505Ab39c9cF94abf69d75acc6115C2E3c0